websites/O/Origin/dist/metadata.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "web",
+    "id": "526087204674600966"
+  },
+  "url": "www.origin.com",
+  "description": {
+    "en": "Play great PC games and connect with your friends, all in one place.",
+    "de": "Spiele großartige PC-Spiele und verbinde dich mit deinen Freunden – alles an einem Ort.",
+    "dk": "Spil gode spil til pc, og opret forbindelse til dine venner – alt sammen fra ét sted.",
+    "es": "Juega a increíbles títulos para PC y conéctate con tus amigos, todo en el mismo lugar.",
+    "fi": "Pelaa upeita PC-pelejä ja pidä yhteyttä ystäviisi yhdessä ja samassa paikassa.",
+    "fr": "Jouez à de grands jeux PC et connectez-vous avec vos amis sur une seule et même plateforme.",
+    "it": "Gioca ai migliori titoli PC e resta in contatto con i tuoi amici, il tutto in un unico spazio.",
+    "jp": "名作PCゲーム。フレンドとのつながり。 その全てはここに。",
+    "ko": "명작 PC 게임, 친구들과의 소통. 한 곳에서 모두 즐기세요!",
+    "nl": "Speel geweldige PC-games en maak verbinding met je vrienden, allemaal op één plek.",
+    "no": "Spill toppspill for PC og kom i kontakt med vennene dine – alt på ett sted.",
+    "pl": "Graj we wspaniałe gry na PC i kontaktuj się ze znajomymi — wszystko w jednym miejscu.",
+    "pt": "Jogue grandes títulos para PC e conecte-se com seus amigos, tudo em um só lugar.",
+    "ru": "Играйте в любимые игры для ПК и общайтесь с друзьями в одном месте.",
+    "sv": "Spela PC-spelen du älskar och umgås med dina vänner, allt på ett enda ställe.",
+    "th": "เล่นเกม PC ชั้นยอดต่างๆ และเชื่อมต่อกับเพื่อนๆ ของคุณได้ในที่เดียว",
+    "zh": "遊玩優質 PC 遊戲並與好友聯繫，一個地方達成所有事。"

```

Comment:
some of the ids arent correct, use these: https://api.premid.app/v2/langFile/list

Reply:
should I do it like JA_JP instead of jp?

Reply:
I should have gotten everything...